build.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+const fs = require('fs');
+const https = require('https');
+const archiver = require('archiver');
+
+https.get('https://uplift.shipit.staging.mozilla-releng.net/coverage/supported_extensions', res => {
+
+  let data = '';
+    
+  res.on('data', chunk => data += chunk);
+  res.on('end', () => {
+    let content = `const SUPPORTED_EXTENSIONS = ${data};`;
+
+    fs.writeFile("supported_extensions.js", content, e => {
+      if (e) {
+        console.log(e.message);
+      }
+    });
+
+    let excludeFiles = [
+      ".gitignore", ".travis.yml", "LICENSE", "README.md",
+      "package.json", "package-lock.json", "build.js"
```

Comment:
Add a comma after "build.js". This way, when we add a new line in the future, the commit will only change one line and not two.